HdrHistogram.Benchmarking/HdrHistogram.Benchmarking.csproj

```diff
@@ -2,14 +2,14 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net47;netcoreapp1.1</TargetFrameworks>
+    <TargetFrameworks>net471;netcoreapp1.1</TargetFrameworks>
```

Comment:
We are not targeting 4.7.1, but still 4.7. I think this would be a breaking change

Reply:
This also breaks the _build.cmd_ file as it does hard code reference the _.\HdrHistogram.Benchmarking\bin\Release\net47\_ :-(